assigner/commands/assign.py

```diff
@@ -64,7 +64,8 @@ def assign(conf, args):
                     repo = StudentRepo.new(base, semester, student_section,
                                            username, token)
                     repo.push(base, branch)
```

Comment:
and repo.push is ok with a list?

Reply:
Yep, it will push multiple branches if you ask it to!